src/style.css

```diff
@@ -1 +1,87 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --font-color: #3D4E61;
+  --border-color: #E1E7ED;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.form__label {
+  display: flex;
+  width: 100%;
+}
+
+.form__input {
+  width: 100%;
+  color: var(--font-color);
+  font-weight: 400;
```

Comment:
Default font-weight must be 300, as specified in the task description. That may be the reason why your tests are broken.